src/v2/stack_aggregator.py

```diff
@@ -381,20 +381,6 @@ def __init__(self, request: StackAggregatorRequest = None,
         self._normalized_packages = normalized_packages
         self.filtered_vul = {}
 
-    def initiate_unknown_package_ingestion(self):
```

Comment:
Is this no longer required?

Reply:
Not required after https://github.com/fabric8-analytics/f8a-server-backbone/pull/266. It will be handled from jobs api.

Reply:
This seems to be added in #266 , are you removing that here?

Reply:
Actually I just deleted the derived class(GoAggregator) implementation because the code is same as base class https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/v2/stack_aggregator.py#L309.

after #266 there is no diff between base class & derived class version of initiate_unknown_package_ingestion.